### ISSUE TITLE
[Discussion] Removing the priority labels

### ISSUE THREAD

**Issue by TheFrenchGhosty** [iv-org/invidious#2198]
This is a discussion mainly for the organization members (@Perflyst @B0pol @cloudrac3r @hackerncoder @saltycrys @SamantazFox @syeopite @unixfox) but others are free to give their opinion:

The priority labels have a ton of problem, mainly that it's inconsistent in the way they are added to issues, the fact that almost nobody understand them (see #1902 ) - it's hard to explain the differences -  and the fact that priorities and bugs are unrelated to priorities on enhancements.

Basically they serve no use and with the creation of https://todo.invidious.io by @unixfox and @SamantazFox it's even more useless.

Removing them is in my opinion the way to go.

**Comment by SamantazFox** [comment]
I'm for their removal.

(And btw, the todo was created by @unixfox, not me ^^)

**Comment by Perflyst** [comment]
Go for it

**Comment by syeopite** [comment]
I'm personally for the priority labels, *however* I feel like the current implementation of them aren't that great. Mainly because of the fact that no one understands them. Though #1902, if resolved, would solve that issue.

And with https://todo.invidious.io, the implementation of them needs to change quite a bit.

The priority labels, at it's core, is a good idea. Developers can see them and know which feature should be worked on first. With the current todo list, there currently isn't any priority. It only tracks the things that needs to be done.

However, some issues are definitely more important than other. #2193 is more important than #2165 for example. This is why I think we should keep the priority labels. Even if we decide to remove them now, I think some sort of priority tracking would come to the todo list in time anyways due to the nature of how some issues are more important than others.

Anyhow, instead of removing them, I think we should actually propagate them to all iv-org repos instead.

**Comment by Computershik73** [comment]
>  than #2165

Without http links to videos, you lose a huge number of old devices. And I wish this to have a high priority.

**Comment by SamantazFox** [comment]
> > than #2165
> 
> Without http links to videos, you lose a huge number of old devices. And I wish this to have a high priority.

There are tons of issues (almost 300) plus many things that we have to do that aren't marked as an issue here in github, but should be done anyway. We have to prioritize what we work on. So unless you provide a PR, feature requests are lower priority than most bugfix, as we prefer having a stable application rather than adding features everywhere and that it becomes unmanageable.

Please understand that, while the amount of devs almost quadrupled since Omarroth has quit the project, we're still a small team of voluteers, and we're only humans. We do as much as we can and try to please everybody :)

**Comment by cloudrac3r** [comment]
The current priority labels don't make sense, but some kind of prioritisation is still important.

**Comment by TheFrenchGhosty** [comment]
Done.
